src/screens/VisualTests/VisualTests.tsx

```diff
@@ -223,13 +247,12 @@ export const VisualTests = ({
   );
 
   const build = getFragment(FragmentBuildFields, data?.build || data?.project?.lastBuild);
+  const isOutdated = build && build.uncommittedHash !== gitInfo.uncommittedHash;
 
   const buildStatusUpdate =
     build &&
-    "tests" in build &&
-    testsToStatusUpdate(getFragment(FragmentTestFields, build.tests.nodes));
-
-  const isOutdated = build && build.uncommittedHash !== gitInfo.uncommittedHash;
+    "testsForStatus" in build &&
+    testsToStatusUpdate(getFragment(FragmentStatusTestFields, build.testsForStatus.nodes));
```

Comment:
It looks like the stories just drop the `testsForStatus` fields and rely on this guard to not break. Are we OK with that? I think maybe it'd be better to include them (and bonus points for assertng `updateBuildStatus` gets called on render in a `play` function).

Reply:
Done and done.